targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC55S69/TARGET_LPCXpresso/PeripheralPinMaps.h

```diff
@@ -19,6 +19,74 @@
 
 #include <mstd_cstddef>
 
+/************GPIO***************/
+MSTD_CONSTEXPR_OBJ_11 PinMap PinMap_GPIO[] = {
```

Comment:
if that's a dummy for testing, should it only be compiled in while building tests?

Reply:
This is only a header file and it looks like this is correct place to add pinmap, even dummy one for testing only.
Maybe we should add an extra comment like here:

https://github.com/ARMmbed/mbed-os/blob/02c5e0806fe0c5fda76be2348c3181363b0f37ec/targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/PeripheralPinMaps.h#L49-L50

Reply:
I think this will also be changed for using `const PinList *pinmap_restricted_gpio_pins()` to exclude GPIO pins, instead of keeping this dummy list. just like what been discussed in the other PR https://github.com/ARMmbed/mbed-os/pull/12380